src/toil_scripts/batch_alignment/bwa_alignment.py

```diff
@@ -222,6 +264,10 @@ def docker_call(work_dir,
     except OSError:
         raise RuntimeError('docker not found on system. Install on all nodes.')
 
+    for filename in outputs.keys():
+        if not os.path.isabs(filename):
```

Comment:
That won't work here because any non-absolute path will be a subdirectory of work_dir, not of the current directory.